src/hooks/index.ts

```diff
@@ -1,10 +1,10 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import {useNetInfo} from '@react-native-community/netinfo'
-import {useFocusEffect} from '@react-navigation/native'
+import {useFocusEffect, useNavigation} from '@react-navigation/native'
```

Comment:
this file shouldn't have any react native stuff in it, out shouldn't have anything yoroi-mobile specific in it